### ISSUE TITLE
Testing Configutation does not build 

### ISSUE THREAD

**Issue by S0S-90** [AKEngels/CAST#31]
When compiling the testing configuration there is an error in the exciton breakup tests, line 113 and 130. It says that there are too many arguments in function call of calculateStartingpoints(). From Travis you can see that the first commit where this happened is 030aa5942c15016140ef4cd02a934375e14f7061

**Comment by mrnicegyu11** [comment]
This seems to be @MichelsBa fault :P Will u fix it? I'd rather not :D

**Comment by S0S-90** [comment]
Furthermore there are some Wrong Order Warnings on Linux that also seem to have something to do with this stuff (at least the word "startingpoint" appears in the warnings). Maybe when you're fixing something there anyway you can also change the order of these variables?

**Comment by S0S-90** [comment]
This was fixed by @MichelsBa, so I closed this issue.

I furthermore fixed the problem that testing configuration did not compile on visual studio with the new premake file where alglib was made a seperate project. It seems that visual studio cannot deal with different debug levels in main project (CAST) and project it is linked to (Alglib) so I had to switch on the debugging symbols for debug and testing configuration in alglib, too.
